Classes/Utility/AlertAction.swift

```diff
@@ -21,6 +21,24 @@ struct AlertAction {
     let title: String?
     let style: UIAlertActionStyle
 
+    enum AlertShareType {
+        case shareUrl
+        case shareContent
+        case shareFilePath
+        case shareFileName
+        case `default`
+
+        var localizedString: String {
+            switch self {
+            case .shareUrl: return NSLocalizedString("Share URL", comment: "")
+            case .shareContent: return NSLocalizedString("Share Content", comment: "")
+            case .shareFilePath: return NSLocalizedString("Copy Path", comment: "")
+            case .shareFileName: return NSLocalizedString("Copy Name", comment: "")
+            default: return NSLocalizedString("Share", comment: "")
```

Comment:
should this be `case .default` instead?

Reply:
I figured it would be a good fallback if there were any other new sharing options to be added.

Reply:
That makes sense. I’ll be making another PR soon for #1958 and can add the change for this in there